File: OneSignalSDK/onesignal/src/main/java/com/onesignal/outcomes/model/OSCachedUniqueOutcome.java
package com.onesignal.outcomes.model;

import com.onesignal.influence.model.OSInfluenceChannel;

public class OSCachedUniqueOutcome {

    private String influenceId;
    private OSInfluenceChannel channel;

    public OSCachedUniqueOutcome(String influenceId, OSInfluenceChannel channel) {
        this.influenceId = influenceId;
        this.channel = channel;
    }

    public String getInfluenceId() {
        return influenceId;
    }

    public OSInfluenceChannel getChannel() {
        return channel;
    }
}

File: OneSignalSDK/onesignal/src/main/java/com/onesignal/outcomes/domain/OSCachedUniqueOutcome.kt
package com.onesignal.outcomes.domain

import com.onesignal.influence.domain.OSInfluenceChannel

open class OSCachedUniqueOutcome(private val influenceId: String,
                                 private val channel: OSInfluenceChannel) {
    open fun getInfluenceId() = influenceId
    open fun getChannel() = channel
}